src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -362,54 +393,96 @@ const LostAndFound = () => {
+          </Grid>
+          <Grid item xs={12} md={10}>
+            {/* Render header row */}
+            {reportHeader()}
+            {/* Filter and display found items */}
+            {foundReports
+              .filter((report) => report.status.toLowerCase() === 'found')
+              .map((report) => reportRow(report, true))}
+          </Grid>
         </Grid>
-        <Grid item xs={12} md={10}>
-          {/* Render header row only on large screens */}
-          {reportHeader()}
-          {/* Active Reports */}
-          {activeReports.map((report) => reportRow(report))}
+      )}
+      {/* Active Missing Item Reports */}
+      {activeReports.length > 0 && (
+        <Grid container justifyContent="center" spacing={3} marginTop={3}>
+          <Grid item xs={12} md={10}>
+            <Card>
+              <CardHeader
+                className="gc360_header"
+                title={
+                  <Typography variant="h5" align="center">
+                    My Active <span className={styles.yellowText}>Lost</span> Item Reports
+                  </Typography>
+                }
+              />
+            </Card>
+          </Grid>
+          <Grid item xs={12} md={10}>
+            {/* Render header row only on large screens */}
+            {reportHeader()}
+            {/* Active Reports */}
+            {activeReports.map((report) => reportRow(report))}
+          </Grid>
         </Grid>
-      </Grid>
+      )}
       <DeleteConfirmationModal
         open={isDeleteModalOpen}
         onClose={handleModalClose}
         onSubmit={handleModalSubmit}
       />
 
       {/* Past Missing Item Reports */}
-      <Grid container justifyContent="center" spacing={3} marginTop={3}>
-        <Grid item xs={12} md={10}>
-          <Card>
-            <CardHeader
-              className="gc360_header"
-              title={
-                <Typography variant="h6" align="left">
-                  My Past Reports
-                </Typography>
-              }
-            />
-          </Card>
-        </Grid>
-        <Grid item xs={12} md={10}>
-          {/* Render header row only on large screens */}
-          {reportHeader(false)}
-          {/* Past Reports */}
-          {pastReports.map((report) => reportRow(report))}
+      {pastReports.length > 0 && (
+        <Grid container justifyContent="center" spacing={3} marginTop={3}>
+          <Grid item xs={12} md={10}>
+            <Card>
+              <CardHeader
+                className="gc360_header"
+                title={
+                  <Typography variant="h6" align="left">
+                    My Past Reports
+                  </Typography>
+                }
+              />
+            </Card>
+          </Grid>
+          <Grid item xs={12} md={10}>
+            {/* Render header row only on large screens */}
+            {reportHeader(false)}
+            {/* Past Reports */}
+            {pastReports.map((report) => reportRow(report))}
```

Comment:
Past reports has the clickable row effect on it again, so it turns blue on hover.  This was something we removed in a previous PR.

<img width="1214" alt="Screen Shot 2024-12-04 at 19 29 34" src="https://github.com/user-attachments/assets/8e731894-80ee-42ee-9ad3-fb516c0e2890">

Reply:
I think thats because that request was merged before i started working on this, that would just be a merge conflict fix

Reply:
Gotcha, I couldn't remember the order, just make sure to test that's the case before you merge to avoid regression.